src/solver_interface.jl

```diff
@@ -521,6 +521,17 @@ each linear constraint.
 """
 function get_farkas_dual! end
 
+"""
+get_farkas_dual_bounds!(m, x::Vector{Float64})
+
+Get the farkas dual (certificate of primal infeasiblility) for the variable
+boundss in the model `m`, and store in `x`. `x`must have one element for
+each variable.
+"""
+function get_farkas_dual_bounds!(m::LinQuadOptimizer, x)
+    nothing
+end
```

Comment:
Woah. Can we make this a separate PR instead of bundling it with the testing change?